kassensichv-client-ios-tests/kassensichv_client_ios_tests.swift

```diff
@@ -0,0 +1,332 @@
+//
+//  kassensichv_client_ios_tests.swift
+//  kassensichv_client_ios_tests
+//
+//  Created by Benjamin Müllner on 07.11.19.
+//  Copyright © 2019 fiskaly. All rights reserved.
+//
+
+import XCTest
+import kassensichv_client_ios
```

Comment:
can we rename that to `FiskalyKassensichvClient` to be consistent with current naming schema (`FiskalyKassensichvSma`)?

Reply:
Renamed it 3b0f6ec